code/mission/missionbriefcommon.cpp

```diff
@@ -284,9 +284,9 @@ void	brief_render_icons(int stage_num, float frametime);
 void	brief_maybe_create_new_grid(grid *gridp, vec3d *pos, matrix *orient, int force = 0);
 grid	*brief_create_grid(grid *gridp, vec3d *forward, vec3d *right, vec3d *center, int nrows, int ncols, float square_size);
 grid	*brief_create_default_grid(void);
-void	brief_render_grid(grid *gridp);
+void	brief_render_grid(grid *gridp, color* color);
```

Comment:
This should also at least take a ``const color*``.
IMO, even here, a const reference, but since this is a function in an old API, it's not as critical.